### PR TITLE
feat: add customer to transaction status response

### PR DESCRIPTION
**Pull request by navoiee**
This PR adds customer object to status response.

### CODE REVIEW

**Comment by navoiee** [comment]
I don't know how to fix coverage so don't mind this atm.

File: Makefile [hunk 1]
```diff
@@ -10,4 +10,4 @@ test:
 setup:
 	go get -v ./...
 	go get github.com/golang/lint/golint
-	go get github.com/fzipp/gocyclo 
+	go get github.com/fzipp/gocyclo
```
Comment (joegrimeringresso) [inline Makefile:13]: :D

File: status.go [hunk 2]
```diff
@@ -4,11 +4,12 @@ import "time"
...
-	PurchaseDatetime     time.Time           `json:"purchase_iso8601_date_and_time"`
-	CurrencyDetails      map[string]Currency `json:"currency_details"`
-	Status               string              `json:"transaction_status"`
+    Languages            []string            `json:"language_list"`
+    MinutesLeftOnReserve float64             `json:"minutes_left_on_reserve"`
+    Trolley              Trolley             `json:"trolley_contents"`
+    ReserveDatetime      time.Time           `json:"reserve_iso8601_date_and_time"`
+    PurchaseDatetime     time.Time           `json:"purchase_iso8601_date_and_time"`
+    CurrencyDetails      map[string]Currency `json:"currency_details"`
+    Customer             Customer            `json:"customer"`
```
Comment (joegrimeringresso) [inline status.go:13]: n.b. The actual change - Jakub likes to bury things in linters

Comment (navoiee) [reply status.go:13]: It's not me - it's because the code is not well formatted ;) But don't worry, I'll fix it ;)

**Review by joegrimeringresso — APPROVED** [review]

**Pull request merged**